tests/zfs-tests/tests/functional/history/cleanup.ksh

```diff
@@ -31,8 +31,11 @@
 
 . $STF_SUITE/include/libtest.shlib
 
-[[ -f $OLD_HISTORY ]] && rm -f $OLD_HISTORY
-[[ -f $TMP_HISTORY ]] && rm -f $TMP_HISTORY
-[[ -f $NEW_HISTORY ]] && rm -f $NEW_HISTORY
+for file in $TEST_BASE_DIR/old_history.* \
```

Comment:
This loop could be simplified to 

`rm -f $TEST_BASE_DIR/{old,tmp,new}_history.*`

Reply:
Indeed it can, I actually like that better so lets go with that.  Thanks!